libs/ui-components/src/components/Device/EditDeviceWizard/steps/ReviewUpdatePolicy.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+
+import { RolloutPolicyForm } from '../../../Fleet/CreateFleet/types';
+import { useTranslation } from '../../../../hooks/useTranslation';
+
+const ReviewUpdatePolicy = ({ rolloutPolicy }: { rolloutPolicy: RolloutPolicyForm }) => {
+  const { t } = useTranslation();
+  if (!rolloutPolicy.isActive) {
+    return '-';
+  }
+
+  return (
+    <Stack hasGutter>
+      <StackItem>{t('{{ count }} batches have been defined', { count: rolloutPolicy.batches.length })}</StackItem>
+    </Stack>
```

Comment:
We dont need to wrap it is Stack, or do we ?

```suggestion
      <>{t('{{ count }} batches have been defined', { count: rolloutPolicy.batches.length })}</>
```

Reply:
True, it wasn't necessary! Removed.